ovos_core/skill_manager.py

```diff
@@ -586,6 +595,14 @@ def _get_skill_loader(self, skill_directory, init_bus=True):
         return SkillLoader(bus, skill_directory)
 
     def _load_skill(self, skill_directory):
+        LOG.warning(f"Found deprecated skill directory: {skill_directory}\n"
+                    f"please create a setup.py for this skill")
+        skill_id = basename(skill_directory)
+        if skill_id in self.blacklist:
+            LOG.warning(f"{skill_id} is blacklisted, it will NOT be loaded")
+            LOG.info(f"Consider uninstalling {skill_id} instead of blacklisting it")
```

Comment:
This message doesn't make sense for a skill that isn't installed, maybe "Consider deleting {skill_directory} instead of blacklisting it"